README.md

```diff
@@ -196,22 +206,58 @@ make bats
 make: 'bats' is up to date.
 ```
 
+`publish` allows the publication of all images targeted by 'linux' to a registry.
+
+`ci-init` is dedicated to Jenkins infrastructure for initialising docker and isn't required in other contexts.
```

Comment:
```suggestion
`buildx-init` is dedicated to Jenkins infrastructure for initializing docker and isn't required in other contexts.
```

Reply:
or

```suggestion
`ci-init` is dedicated to Jenkins infrastructure for initializing docker and isn't required in other contexts.
```

Reply:
I chose the English spelling: https://sapling.ai/usage/initialize-vs-initialise, is that OK?

Reply:
https://github.com/jenkinsci/docker-ssh-agent/pull/421#pullrequestreview-2124965521